### PR TITLE
chore(pt-br): replace old-style specification tables part 7

### PR DESCRIPTION
**Pull request by yin1999**
### Description

- replace old-style specification tables
- replace `{{SpecName}}` macro calls

### Related issues and pull requests

continue: #16454

### CODE REVIEW

File: files/pt-br/glossary/css_selector/index.md [hunk 1]
```diff
@@ -50,4 +55,4 @@ Nessa primeira regra do CSS, eu estou selecionando o elemento da div estilizando
 
 ### Referência técnica
 
-{{SpecName("CSS3 Selectors")}}
+[Selectors Level 3](https://drafts.csswg.org/selectors-3/)
```
Comment (yin1999) [inline files/pt-br/glossary/css_selector/index.md:58]: If the macro call is not in a specification table, just replace it with the corresponding url. For the rest of content, follow the rule

File: files/pt-br/web/javascript/reference/global_objects/function/arguments/index.md [hunk 2]
```diff
@@ -45,12 +45,7 @@ console.log("returned: " + g.arguments);
...
-| -------------------------------------------------------------------- | -------------------- | ------------------------------------------------------------------------------------------------------------------------------ |
-| {{SpecName('ES1')}}                                                  | {{Spec2('ES1')}}     | Definiçao inicial. Implementado em JavaScript 1.0. Obsoleto em favor de {{jsxref("Functions/arguments", "arguments")}} em ES3. |
-| {{SpecName('ES5.1', '#sec-10.6', 'arguments object')}}               | {{Spec2('ES5.1')}}   | {{jsxref("Functions/arguments", "arguments")}} object                                                                          |
-| {{SpecName('ES6', '#sec-arguments-object', 'arguments object')}}     | {{Spec2('ES6')}}     | {{jsxref("Functions/arguments", "arguments")}} object                                                                          |
-| {{SpecName('ESDraft', '#sec-arguments-object', 'arguments object')}} | {{Spec2('ESDraft')}} | {{jsxref("Functions/arguments", "arguments")}} object                                                                          |
+Not part of any standard.
```
Comment (yin1999) [inline files/pt-br/web/javascript/reference/global_objects/function/arguments/index.md:48]: It's better to localize this, but I can't speak brazilian portuguese. Feel free to modify this.

Comment (josielrocha) [inline files/pt-br/web/javascript/reference/global_objects/function/arguments/index.md:48]:
```suggestion
Não faz parte de nenhuma especificação.
```

File: files/pt-br/web/javascript/reference/global_objects/array/foreach/index.md [hunk 3]
```diff
@@ -172,11 +172,7 @@ if (!Array.prototype.forEach) {
 
 ## Specifications
```
Comment (queengooborg) [inline files/pt-br/web/javascript/reference/global_objects/array/foreach/index.md:173]:
```suggestion
## Especificaçãoes
```

File: files/pt-br/web/javascript/reference/global_objects/date/parse/index.md [hunk 4]
```diff
@@ -133,11 +133,7 @@ Returns `14400000` no matter the local time zone as a time zone GMT (UTC) is pro
 
 ## Specifications
```
Comment (queengooborg) [inline files/pt-br/web/javascript/reference/global_objects/date/parse/index.md:134]:
```suggestion
## Especificaçãoes
```

File: files/pt-br/web/javascript/reference/global_objects/intl/index.md [hunk 5]
```diff
@@ -57,11 +57,7 @@ One property is supported by all language sensitive constructors and functions:
 
 ## Specifications
```
Comment (queengooborg) [inline files/pt-br/web/javascript/reference/global_objects/intl/index.md:58]:
```suggestion
## Especificaçãoes
```

File: files/pt-br/web/javascript/reference/global_objects/isnan/index.md [hunk 6]
```diff
@@ -85,9 +85,7 @@ You can use this, for example, to test whether an argument to a function is arit
 
 ## Specifications
```
Comment (queengooborg) [inline files/pt-br/web/javascript/reference/global_objects/isnan/index.md:86]:
```suggestion
## Especificaçãoes
```

File: files/pt-br/web/javascript/reference/global_objects/math/atan2/index.md [hunk 7]
```diff
@@ -57,12 +57,7 @@ Math.atan2(±Infinity, +Infinity); // ±PI/4.
 
 ## Specifications
```
Comment (queengooborg) [inline files/pt-br/web/javascript/reference/global_objects/math/atan2/index.md:58]:
```suggestion
## Especificaçãoes
```

File: files/pt-br/web/javascript/reference/global_objects/math/expm1/index.md [hunk 8]
```diff
@@ -52,9 +52,7 @@ Math.expm1(1); // 1.718281828459045
 
 ## Specifications
```
Comment (queengooborg) [inline files/pt-br/web/javascript/reference/global_objects/math/expm1/index.md:53]:
```suggestion
## Especificaçãoes
```

File: files/pt-br/web/javascript/reference/global_objects/math/tanh/index.md [hunk 9]
```diff
@@ -56,10 +56,7 @@ Math.tanh =
 
 ## Specifications
```
Comment (queengooborg) [inline files/pt-br/web/javascript/reference/global_objects/math/tanh/index.md:57]:
```suggestion
## Especificaçãoes
```

File: files/pt-br/web/javascript/reference/global_objects/number/valueof/index.md [hunk 10]
```diff
@@ -38,12 +38,7 @@ console.log(typeof num); // number
 
 ## Specifications
```
Comment (queengooborg) [inline files/pt-br/web/javascript/reference/global_objects/number/valueof/index.md:39]:
```suggestion
## Especificaçãoes
```

File: files/pt-br/web/javascript/reference/global_objects/object/isfrozen/index.md [hunk 11]
```diff
@@ -132,11 +132,7 @@ Object.isFrozen(1);
 
 ## Specifications
```
Comment (queengooborg) [inline files/pt-br/web/javascript/reference/global_objects/object/isfrozen/index.md:133]:
```suggestion
## Especificaçãoes
```

File: files/pt-br/web/javascript/reference/global_objects/object/isprototypeof/index.md [hunk 12]
```diff
@@ -79,12 +79,7 @@ if (Foo.prototype.isPrototypeOf(baz)) {
 
 ## Specificações
```
Comment (queengooborg) [inline files/pt-br/web/javascript/reference/global_objects/object/isprototypeof/index.md:80]:
```suggestion
## Especificaçãoes
```

File: files/pt-br/web/javascript/reference/global_objects/reflect/construct/index.md [hunk 13]
```diff
@@ -114,10 +114,7 @@ d.getFullYear(); // 1776
 
 ## Specifications
```
Comment (queengooborg) [inline files/pt-br/web/javascript/reference/global_objects/reflect/construct/index.md:115]:
```suggestion
## Especificaçãoes
```

File: files/pt-br/web/javascript/reference/global_objects/regexp/test/index.md [hunk 14]
```diff
@@ -57,11 +57,7 @@ function testinput(re, str) {
 
 ## Specificações
```
Comment (queengooborg) [inline files/pt-br/web/javascript/reference/global_objects/regexp/test/index.md:58]:
```suggestion
## Especificaçãoes
```

File: files/pt-br/web/javascript/reference/global_objects/unescape/index.md [hunk 15]
```diff
@@ -36,12 +36,7 @@ unescape("%u0107"); // "ć"
 
 ## Specifications
```
Comment (queengooborg) [inline files/pt-br/web/javascript/reference/global_objects/unescape/index.md:37]:
```suggestion
## Especificaçãoes
```

File: files/pt-br/web/javascript/reference/operators/index.md [hunk 16]
```diff
@@ -213,11 +213,7 @@ An assignment operator assigns a value to its left operand based on the value of
 
 ## Specifications
```
Comment (queengooborg) [inline files/pt-br/web/javascript/reference/operators/index.md:214]:
```suggestion
## Especificaçãoes
```

File: files/pt-br/web/javascript/reference/operators/void/index.md [hunk 17]
```diff
@@ -54,11 +54,7 @@ Note que no entanto o pseudo protocolo `javascript:` tem seu uso desencorajado p
 
 ## Specificações
```
Comment (queengooborg) [inline files/pt-br/web/javascript/reference/operators/void/index.md:55]:
```suggestion
## Especificaçãoes
```

File: files/pt-br/web/javascript/reference/statements/for...in/index.md [hunk 18]
```diff
@@ -82,13 +82,7 @@ for (var prop in obj) {
 
 ## Specificações
```
Comment (queengooborg) [inline files/pt-br/web/javascript/reference/statements/for...in/index.md:83]:
```suggestion
## Especificaçãoes
```

File: files/pt-br/web/javascript/reference/statements/throw/index.md [hunk 19]
```diff
@@ -162,12 +162,7 @@ try {
 
 ## Specifications
```
Comment (queengooborg) [inline files/pt-br/web/javascript/reference/statements/throw/index.md:163]:
```suggestion
## Especificaçãoes
```

File: files/pt-br/web/svg/attribute/keytimes/index.md [hunk 20]
```diff
@@ -77,10 +77,7 @@ If the duration of the animation is indefinite, any `keyTimes` specification wil
 
 ## Specifications
```
Comment (queengooborg) [inline files/pt-br/web/svg/attribute/keytimes/index.md:78]:
```suggestion
## Especificaçãoes
```

File: files/pt-br/web/svg/attribute/preserveaspectratio/index.md [hunk 21]
```diff
@@ -489,8 +489,4 @@ For {{SVGElement('view')}}, `preserveAspectRatio` indicates if a uniform scaling
 
 ## Specification
```
Comment (queengooborg) [inline files/pt-br/web/svg/attribute/preserveaspectratio/index.md:490]:
```suggestion
## Especificaçãoes
```

File: files/pt-br/web/svg/element/a/index.md [hunk 22]
```diff
@@ -114,10 +114,7 @@ svg|a:active {
 
 ## Specifications
```
Comment (queengooborg) [inline files/pt-br/web/svg/element/a/index.md:115]:
```suggestion
## Especificaçãoes
```

File: files/pt-br/web/svg/element/animate/index.md [hunk 23]
```diff
@@ -71,10 +71,7 @@ Consider providing a mechanism for pausing or disabling animation, as well as us
 
 ## Specifications
```
Comment (queengooborg) [inline files/pt-br/web/svg/element/animate/index.md:72]:
```suggestion
## Especificaçãoes
```

File: files/pt-br/web/svg/element/svg/index.md [hunk 24]
```diff
@@ -92,10 +92,7 @@ svg {
 
 ## Specifications
```
Comment (queengooborg) [inline files/pt-br/web/svg/element/svg/index.md:93]:
```suggestion
## Especificaçãoes
```

**Review by queengooborg — CHANGES_REQUESTED** [review]
Just some tweaks to the headers, LGTM otherwise!

File: files/pt-br/web/html/element/track/index.md [hunk 25]
```diff
@@ -116,10 +116,7 @@ A `media` element cannot have more than one `track` with the same `kind`, `srcla
 
 ## Specifications
```
[the remark this answers is not in the document]
Comment (yin1999) [reply files/pt-br/web/html/element/track/index.md:117]: I'm not really sure, should we use "Especificacões"?

[the remark this answers is not in the document]
Comment (josielrocha) [reply files/pt-br/web/html/element/track/index.md:117]: Hi @yin1999, @queengooborg, thank you for your contributions!

We should use `Especificações`, with `ç`

[the remark this answers is not in the document]
Comment (yin1999) [reply files/pt-br/web/html/element/track/index.md]: Updated!

**Review by queengooborg — APPROVED** [review]
LGTM, thank you!

Note to other reviewers: the failing test is due to existing content and not new changes.

**Pull request merged**